app/src/main/java/com/sma2/sma2/DataAccess/ExerciseDataService.java

```diff
@@ -0,0 +1,65 @@
+package com.sma2.sma2.DataAccess;
+
+import android.content.Context;
+
+import com.sma2.sma2.ExerciseLogic.Exercise;
+
+import org.greenrobot.greendao.database.Database;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ExerciseDataService {
+    private Context invocationcontext;
+    public ExerciseDataService(Context context){
+        this.invocationcontext = context;
+    }
+
+    public void saveExercise(Exercise exercise){
+        ExerciseDA tmp = new ExerciseDA(exercise);
+        DaoMaster.DevOpenHelper helper = new DaoMaster.DevOpenHelper(invocationcontext, "apkinsondb");
```

Comment:
This is fine for now, but we should refactor this to remove redundant code

Reply:
We will have to refactor at some point. But for now, lets get things done quick and dirty